### PR TITLE
Introduce anylogger to SwingSet and cosmic-swingset

### LINKED ISSUE #571: agoric start output is super noisy
<details>
   <summary>Console output for working "agoric start"</summary>
<pre>$ agoric start
initializing dev
/home/david/projects/agoric-sdk/packages/cosmic-swingset/bin/ag-solo init dev --egresses=fake
morgan deprecated default format: use combined format ../../node_modules/esm/esm.js:1:278827
solo [ 'init', 'dev', '--egresses=fake' ]
ag-solo initialized in dev
HTTP/WebSocket will listen on 127.0.0.1:8000
setting fake chain with 5 second delay
/home/david/projects/agoric-sdk/packages/cosmic-swingset/bin/ag-solo set-fake-chain --role=two_chain --delay=5 myFakeGCI
morgan deprecated default format: use combined format ../../../node_modules/esm/esm.js:1:278827
solo [ 'set-fake-chain', '--role=two_chain', '--delay=5', 'myFakeGCI' ]
linking html directories
/home/david/projects/agoric-sdk/packages/cosmic-swingset/bin/ag-solo start --role=two_client
morgan deprecated default format: use combined format ../../../node_modules/esm/esm.js:1:278827
solo [ 'start', '--role=two_client' ]
adding HTTP/WS listener on 127.0.0.1:8000
Serving static files from /home/david/projects/agoric-sdk/demo/.agservers/dev/html
adding follower/sender for fake chain two_chain myFakeGCI
Listening on 127.0.0.1:8000
launch: checking for saved mailbox state false
initializing empty swingset state
buildSwingset
kernel.addDevice(mailbox)
kernel.addDevice(timer)
= adding vat 'exchange' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-exchange.js
= adding vat 'host' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-host.js
= adding vat 'http' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-http.js
= adding vat 'mints' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-mints.js
= adding vat 'provisioning' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-provisioning.js
= adding vat 'registrar' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-registrar.js
= adding vat 'sharing' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-sharing.js
= adding vat 'uploads' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-uploads.js
= adding vat 'wallet' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-wallet.js
= adding vat 'zoe' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-zoe.js
= adding vat 'vattp' from /home/david/projects/agoric-sdk/packages/SwingSet/src/vats/vat-tp.js
= adding vat 'comms' from /home/david/projects/agoric-sdk/packages/SwingSet/src/vats/comms/index.js
= adding vat 'timer' from /home/david/projects/agoric-sdk/packages/SwingSet/src/vats/vat-timerWrapper.js
= adding vat '_bootstrap' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/bootstrap.js
loading bootstrap.js
wasInitialized = false
Assigned VatID v1 for genesis vat exchange
Assigned VatID v2 for genesis vat host
Assigned VatID v3 for genesis vat http
Assigned VatID v4 for genesis vat mints
Assigned VatID v5 for genesis vat provisioning
Assigned VatID v6 for genesis vat registrar
Assigned VatID v7 for genesis vat sharing
Assigned VatID v8 for genesis vat uploads
Assigned VatID v9 for genesis vat wallet
Assigned VatID v10 for genesis vat zoe
Assigned VatID v11 for genesis vat vattp
Assigned VatID v12 for genesis vat comms
Assigned VatID v13 for genesis vat timer
Assigned VatID v14 for genesis vat _bootstrap
Assigned VatID v15 for genesis vat vatAdmin
Assigned DeviceID d7 for genesis device mailbox
Assigned DeviceID d8 for genesis device timer
Assigned DeviceID d9 for genesis device vatAdmin
=> queueing bootstrap()
adding vref _bootstrap [v14]
adding vref comms [v12]
adding vref exchange [v1]
adding vref host [v2]
adding vref http [v3]
adding vref mints [v4]
adding vref provisioning [v5]
adding vref registrar [v6]
adding vref sharing [v7]
adding vref timer [v13]
adding vref uploads [v8]
adding vref vatAdmin [v15]
adding vref vattp [v11]
adding vref wallet [v9]
adding vref zoe [v10]
adding dref mailbox [d7]
adding dref timer [d8]
adding dref vatAdmin [d9]
two_chain bootstrap starting
all vats initialized for two_chain
= loading config from basedir /home/david/projects/agoric-sdk/demo/.agservers/dev/vats
ignoring  bootstrap.js
ignoring  gci.js
ignoring  lib-exchange.js
ignoring  lib-wallet.js
ignoring  observable.js
ignoring  pubsub.js
ignoring  repl.js
ignoring  scratch.js
kernel.addDevice(mailbox)
kernel.addDevice(command)
kernel.addDevice(timer)
= adding vat 'exchange' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-exchange.js
= adding vat 'host' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-host.js
= adding vat 'http' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-http.js
= adding vat 'mints' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-mints.js
= adding vat 'provisioning' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-provisioning.js
= adding vat 'registrar' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-registrar.js
= adding vat 'sharing' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-sharing.js
= adding vat 'uploads' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-uploads.js
= adding vat 'wallet' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-wallet.js
= adding vat 'zoe' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/vat-zoe.js
= adding vat 'vattp' from /home/david/projects/agoric-sdk/packages/SwingSet/src/vats/vat-tp.js
= adding vat 'comms' from /home/david/projects/agoric-sdk/packages/SwingSet/src/vats/comms/index.js
= adding vat 'timer' from /home/david/projects/agoric-sdk/packages/SwingSet/src/vats/vat-timerWrapper.js
= adding vat '_bootstrap' from /home/david/projects/agoric-sdk/demo/.agservers/dev/vats/bootstrap.js
loading bootstrap.js
wasInitialized = false
Assigned VatID v1 for genesis vat exchange
Assigned VatID v2 for genesis vat host
Assigned VatID v3 for genesis vat http
Assigned VatID v4 for genesis vat mints
Assigned VatID v5 for genesis vat provisioning
Assigned VatID v6 for genesis vat registrar
Assigned VatID v7 for genesis vat sharing
Assigned VatID v8 for genesis vat uploads
Assigned VatID v9 for genesis vat wallet
Assigned VatID v10 for genesis vat zoe
Assigned VatID v11 for genesis vat vattp
Assigned VatID v12 for genesis vat comms
Assigned VatID v13 for genesis vat timer
Assigned VatID v14 for genesis vat _bootstrap
Assigned VatID v15 for genesis vat vatAdmin
Assigned DeviceID d7 for genesis device mailbox
Assigned DeviceID d8 for genesis device command
Assigned DeviceID d9 for genesis device timer
Assigned DeviceID d10 for genesis device vatAdmin
=> queueing bootstrap()
adding vref _bootstrap [v14]
adding vref comms [v12]
adding vref exchange [v1]
adding vref host [v2]
adding vref http [v3]
adding vref mints [v4]
adding vref provisioning [v5]
adding vref registrar [v6]
adding vref sharing [v7]
adding vref timer [v13]
adding vref uploads [v8]
adding vref vatAdmin [v15]
adding vref vattp [v11]
adding vref wallet [v9]
adding vref zoe [v10]
adding dref command [d8]
adding dref mailbox [d7]
adding dref timer [d9]
adding dref vatAdmin [d10]
two_client bootstrap starting
deliver { myFakeGCI: { outbox: [ [Array] ], inboundAck: 0 } }
new outbound message 1 for myFakeGCI: deliver:ro+1:getDemoBundle:rp-40;["client"]
 1 new messages
 invoking deliverator
swingset running
polled; blockTime:1582108259, h:0 ADDED: false
wrote SwingSet checkpoint (kernel=17179, mailbox=2), [run=0ms, mb=0ms, save=15ms]
mboxDeliver:   ADDED messages
inbound promise rp-40 mapped to p+20
comms.notifyFulfillToData(p+20)
outbox changed
wrote SwingSet checkpoint (kernel=26776, mailbox=694), [run=60ms, mb=0ms, save=18ms]
deliver {
  myFakeGCI: { outbox: [ [Array], [Array], [Array], [Array] ], inboundAck: 1 }
}
new outbound message 2 for myFakeGCI: deliver:ro+22:getLabel:rp-41;[]
new outbound message 3 for myFakeGCI: deliver:ro+22:getExtentOps:rp-42;[]
new outbound message 4 for myFakeGCI: deliver:ro+23:getLabel:rp-43;[]
new outbound message 5 for myFakeGCI: deliver:ro+23:getExtentOps:rp-44;[]
 4 new messages
 invoking deliverator
polled; blockTime:1582108264, h:1 ADDED: false
wrote SwingSet checkpoint (kernel=26776, mailbox=694), [run=0ms, mb=0ms, save=17ms]
mboxDeliver:   ADDED messages
inbound promise rp-41 mapped to p+21
inbound promise rp-42 mapped to p+22
inbound promise rp-43 mapped to p+23
inbound promise rp-44 mapped to p+24
comms.notifyFulfillToData(p+21)
comms.notifyFulfillToData(p+22)
comms.notifyFulfillToData(p+23)
comms.notifyFulfillToData(p+24)
outbox changed
wrote SwingSet checkpoint (kernel=36760, mailbox=389), [run=69ms, mb=1ms, save=13ms]
deliver { myFakeGCI: { outbox: [ [Array], [Array] ], inboundAck: 5 } }
new outbound message 6 for myFakeGCI: deliver:ro+22:makeEmptyPurse:rp-45;["purse"]
new outbound message 7 for myFakeGCI: deliver:ro+23:makeEmptyPurse:rp-46;["purse"]
 2 new messages
 invoking deliverator
polled; blockTime:1582108269, h:2 ADDED: false
wrote SwingSet checkpoint (kernel=36760, mailbox=389), [run=0ms, mb=1ms, save=21ms]
mboxDeliver:   ADDED messages
inbound promise rp-45 mapped to p+25
inbound promise rp-46 mapped to p+26
comms.notifyFulfillToPresence(p+25) = o-62
comms.notifyFulfillToPresence(p+26) = o-63
outbox changed
wrote SwingSet checkpoint (kernel=41787, mailbox=116), [run=34ms, mb=0ms, save=21ms]
subscribing to walletPurseState
subscribing to walletInboxState
all vats initialized for two_client
deliver {
  myFakeGCI: {
    outbox: [ [Array], [Array], [Array], [Array], [Array], [Array] ],
    inboundAck: 7
  }
}
new outbound message 8 for myFakeGCI: deliver:ro+29:getBalance:rp-47;[]
new outbound message 9 for myFakeGCI: deliver:ro+29:getAssay:rp-48;[]
new outbound message 10 for myFakeGCI: deliver:ro+30:getBalance:rp-49;[]
new outbound message 11 for myFakeGCI: deliver:ro+30:getAssay:rp-50;[]
new outbound message 12 for myFakeGCI: deliver:ro+29:depositAll:rp-51:ro+20;[{"@qclass":"slot","index":0}]
new outbound message 13 for myFakeGCI: deliver:ro+30:depositAll:rp-52:ro+21;[{"@qclass":"slot","index":0}]
 6 new messages
 invoking deliverator
polled; blockTime:1582108274, h:3 ADDED: false
wrote SwingSet checkpoint (kernel=41787, mailbox=116), [run=0ms, mb=1ms, save=15ms]
mboxDeliver:   ADDED messages
inbound promise rp-47 mapped to p+27
inbound promise rp-48 mapped to p+28
inbound promise rp-49 mapped to p+29
inbound promise rp-50 mapped to p+30
inbound promise rp-51 mapped to p+31
inbound promise rp-52 mapped to p+32
comms.notifyFulfillToData(p+27)
comms.notifyFulfillToPresence(p+28) = o-55
comms.notifyFulfillToData(p+29)
comms.notifyFulfillToPresence(p+30) = o-56
comms.notifyFulfillToData(p+31)
comms.notifyFulfillToData(p+32)
outbox changed
wrote SwingSet checkpoint (kernel=57984, mailbox=649), [run=97ms, mb=1ms, save=12ms]
deliver {
  myFakeGCI: { outbox: [ [Array], [Array], [Array], [Array] ], inboundAck: 13 }
}
new outbound message 14 for myFakeGCI: deliver:ro+29:getBalance:rp-53;[]
new outbound message 15 for myFakeGCI: deliver:ro+29:getAssay:rp-54;[]
new outbound message 16 for myFakeGCI: deliver:ro+30:getBalance:rp-55;[]
new outbound message 17 for myFakeGCI: deliver:ro+30:getAssay:rp-56;[]
 4 new messages
 invoking deliverator
polled; blockTime:1582108279, h:4 ADDED: false
wrote SwingSet checkpoint (kernel=57984, mailbox=649), [run=0ms, mb=0ms, save=18ms]
mboxDeliver:   ADDED messages
inbound promise rp-53 mapped to p+33
inbound promise rp-54 mapped to p+34
inbound promise rp-55 mapped to p+35
inbound promise rp-56 mapped to p+36
comms.notifyFulfillToData(p+33)
comms.notifyFulfillToPresence(p+34) = o-55
comms.notifyFulfillToData(p+35)
comms.notifyFulfillToPresence(p+36) = o-56
outbox changed
wrote SwingSet checkpoint (kernel=68596, mailbox=388), [run=72ms, mb=0ms, save=14ms]
deliver { myFakeGCI: { outbox: [], inboundAck: 17 } }
 0 new messages
 invoking deliverator
polled; blockTime:1582108284, h:5 ADDED: false
wrote SwingSet checkpoint (kernel=68596, mailbox=388), [run=0ms, mb=0ms, save=24ms]
mboxDeliver:   ADDED messages
outbox changed
wrote SwingSet checkpoint (kernel=69449, mailbox=50), [run=6ms, mb=1ms, save=11ms]
polled; blockTime:1582108289, h:6 ADDED: false
wrote SwingSet checkpoint (kernel=69449, mailbox=50), [run=0ms, mb=0ms, save=27ms]
polled; blockTime:1582108295, h:7 ADDED: false
wrote SwingSet checkpoint (kernel=69449, mailbox=50), [run=0ms, mb=0ms, save=19ms]
polled; blockTime:1582108300, h:8 ADDED: false
wrote SwingSet checkpoint (kernel=69449, mailbox=50), [run=0ms, mb=0ms, save=21ms]
polled; blockTime:1582108305, h:9 ADDED: false
wrote SwingSet checkpoint (kernel=69449, mailbox=50), [run=0ms, mb=0ms, save=21ms]
polled; blockTime:1582108310, h:10 ADDED: false
wrote SwingSet checkpoint (kernel=69449, mailbox=50), [run=0ms, mb=0ms, save=13ms]

(the last lines repeat indefinitely)
</pre>
</details>

Instant reaction:
nopetopus

---

I understand all of this is probably useful for debugging purposes, but could it be hidden behind an opt-in (like a `--debug` flag)?

I would have expected something like (intention, not real content):
```sh
$ agoric start
✓ Local Agoric Node started
✓ Local Wallet server started, ready to receive requests at http://localhost:5555/
✓ Local Dapp server started, ready to receive requests at http://localhost:8888/

Everything is ok and ready!
Keep this terminal open somewhere
In order to deploy a contract, you need to open a new terminal
```

### PR DESCRIPTION
**Pull request by michaelfig**
Closes #571

In a continued effort to reduce confusing information as part of agoric-cli and the ag-chain-cosmos logs, I've introduced [`anylogger`](https://github.com/Download/anylogger) to the SwingSet and cosmic-swingset libraries.  This is based on my positive experience with anylogger in agoric-cli.

Notably, I've tagged the SwingSet anylogger instances with the name of the vat, device, or kernel, and passed it in as the `console` endowment to the ses-evaluated things.

This doesn't change any existing functionality (by default, anylogger just uses console.log and friends unchanged) but makes it possible for host programs to redirect, enhance, or filter out the log messages.

The magic happens in `ag-solo` and `ag-chain-cosmos` where they import `anylogger-agoric.js` to modify anylogger.  That makes anylogger sensitive to `export DEBUG=agoric`, and if it is not set, to log only the `.info`, `.warn`, and `.error` levels.  If that environment variable is set, then you get the full dumps, prefixed (approximately) by the `TAG` supplied to the `const log =. anylogger(TAG)`.  I find this truly useful when debugging.

In the future, we can change this policy without having to rewrite any library code or other modules except for `cosmic-swingset/lib/anylogger-agoric.js`.

### CODE REVIEW

File: packages/SwingSet/src/controller.js [hunk 1]
```diff
@@ -254,7 +263,7 @@ export async function buildVatController(config, withSES = true, argv = []) {
   // Evaluate source to produce a setup function. This binds withSES from the
   // enclosing context and evaluates it either in a SES context, or without SES
   // by directly calling require().
-  async function evaluateToSetup(sourceIndex, filePrefix = undefined) {
+  async function evaluateToSetup(sourceIndex, tag = undefined) {
```
Comment (FUDCo) [inline packages/SwingSet/src/controller.js:266]: Some of this looks like it's not anylogger related at all.  Am I misunderstanding?

File: packages/SwingSet/src/kernel/liveSlots.js [hunk 2]
```diff
@@ -276,8 +276,16 @@ function build(syscall, _state, makeRoot, forVatID) {
...
+      // We return the results of the resolve/reject, rather
+      // than rejecting the dispatch with whatever the method
+      // did.
+      // It is up to the caller to handle or fail to handle the
+      // rejection.  Failing to handle should trigger a platform
+      // log, rather than via our parent doProcess call.
+
       // eslint-disable-next-line no-use-before-define
-      p.then(thenResolve(result), thenReject(result));
+      return p.then(thenResolve(result), thenReject(result));
```
Comment (Chris-Hibbert) [inline packages/SwingSet/src/kernel/liveSlots.js]: Does this belong in a different PR?

File: packages/cosmic-swingset/lib/ag-solo/main.js [hunk 3]
```diff
@@ -66,7 +70,7 @@ start
     assert(basedir !== undefined, 'you must provide a BASEDIR');
     initBasedir(basedir, webport, webhost, subdir, egresses.split(','));
     await resetState(basedir);
-    // console.error(
+    // log.error(
```
Comment (Chris-Hibbert) [inline packages/cosmic-swingset/lib/ag-solo/main.js:76]: uncomment or drop

File: packages/SwingSet/src/controller.js (hunk 1 above)
Comment (michaelfig) [reply packages/SwingSet/src/controller.js:266]: > Some of this looks like it's not anylogger related at all. Am I misunderstanding?

I'm configuring the `console` endowment to be redirected to anylogger.  Changing the tag will allow for filtering based on vat, etc (in the future).  The `filePrefix` was a little too low-level, instead it should be just a tag out of which the `filePrefix` is built.

File: packages/cosmic-swingset/lib/ag-solo/start.js [hunk 4]
```diff
@@ -282,7 +286,7 @@ export default async function start(basedir, withSES, argv) {
   // Start timer here!
   startTimer(1200);
 
-  console.log(`swingset running`);
+  log.info(`swingset running`);
```
Comment (Chris-Hibbert) [inline packages/cosmic-swingset/lib/ag-solo/start.js:289]: This one might be helpful. Does anything come later to tell you it's running? I'd make it 
```
log('swingset running');
```

File: packages/cosmic-swingset/lib/block-manager.js [hunk 5]
```diff
@@ -126,7 +130,7 @@ export default function makeBlockManager({
     savedActions = currentActions;
     computedHeight = action.blockHeight;
 
-    console.log(
+    log(
```
Comment (Chris-Hibbert) [inline packages/cosmic-swingset/lib/block-manager.js:133]: maybe log.info()?  Does anylogger give you the ability to throttle? [e.g. I want to see every 20th message from this source.]

**Review by Chris-Hibbert — APPROVED** [review]
looks fine. I applaud the goal of better and less visible logging.

File: packages/cosmic-swingset/lib/ag-solo/start.js (hunk 4 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/start.js:289]: `log.info()` is higher priority than `log()`.

File: packages/cosmic-swingset/lib/block-manager.js (hunk 5 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/block-manager.js:133]: As above.

For your second question, you're welcome to implement that in `packages/cosmic-swingset/lib/anylogger-agoric.js`. 😉

File: packages/cosmic-swingset/lib/ag-solo/main.js (hunk 3 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/main.js:76]: It's code that used to exist, and we might need it again.  So, I'll leave it commented-out and add a `TODO`.

File: packages/SwingSet/src/kernel/liveSlots.js (hunk 2 above)
Comment (michaelfig) [reply packages/SwingSet/src/kernel/liveSlots.js]: Yup!

Comment (warner) [reply packages/SwingSet/src/kernel/liveSlots.js]: that's the change that makes the kernel (well, liveslots) stop announcing vat promise rejections, right?

I guess if we're generally ok with teaching developers to use the new logging tools to hear about these, then I'm ok with this landing now. I opened #726 to discuss it, but I think we didn't raise any objections when talking about it this afternoon.

File: packages/SwingSet/src/controller.js [hunk 6]
```diff
@@ -199,13 +203,18 @@ function realmRegisterEndOfCrank(fn) {
...
+    const filePrefix = `/SwingSet/${tag}`;
+    const localLog = anylogger(`SwingSet:${tag}`);
+
     const nestedEvaluate = source =>
       s.evaluate(source, {
         // Support both getExport and nestedEvaluate module format.
         require: r,
         nestedEvaluate,
 
+        console: harden(localLog),
```
Comment (warner) [inline packages/SwingSet/src/controller.js:216]: Can we audit (or at least skim) the `anylogger` code to make sure this doesn't provide a communication channel between otherwise isolated objects in the vat? We might need to put a wrapper, ala `makeConsole.js`, around the `localLog` to prevent access to mutable state which could be used as a channel.

File: packages/SwingSet/src/kernel/liveSlots.js (hunk 2 above)
Comment (michaelfig) [reply packages/SwingSet/src/kernel/liveSlots.js]: Already separated out into #732

File: packages/cosmic-swingset/lib/ag-chain-cosmos [hunk 7]
```diff
@@ -4,6 +4,11 @@ const path = require('path');
 const agcc = require('bindings')('agcosmosdaemon.node');
 const esmRequire = require('esm')(module);
 
+esmRequire('./anylogger-agoric');
+const anylogger = require('anylogger');
```
Comment (warner) [inline packages/cosmic-swingset/lib/ag-chain-cosmos:8]: Why use `esmRequire` for some and `require` for others? I know you changed this earlier to retain the original (non-ESM-enabled) `require`, but is there an advantage to eschewing the `esmRequire` form?

Also, why aren't we importing anything from `anylogger-agoric`.. is it only doing side-effect-ish manipulation of something that is used by the subsequent `anylogger` import?

File: packages/cosmic-swingset/lib/ag-solo/web.js [hunk 8]
```diff
@@ -140,22 +146,19 @@ export function makeHTTPListener(basedir, port, host, rawInboundCommand) {
...
+      log.info(id, 'client closed');
       broadcasts.delete(ws);
       if (req.url === '/captp') {
-        inboundCommand(
-          { type: 'CTP_CLOSE', connectionID },
-          req,
-          id,
-        ).catch(_ => {});
-        points.delete(connectionID);
+        inboundCommand({ type: 'CTP_CLOSE', connectionID }, req, id)
```
Comment (warner) [inline packages/cosmic-swingset/lib/ag-solo/web.js:159]: This looks like an unrelated (but possibly correct) change to the ordering of `points.delete` and the command handler.. maybe put it in a separate cleanup PR? It's not immediately clear to me that we should defer removing it until after the command handler finishes.

**Review by warner — APPROVED** [review]
Please take a look at the issues I raised (particularly `anylogger` as a communications channel), but if you're comfortable with the answers, go ahead and land.

File: packages/SwingSet/src/controller.js (hunk 6 above)
Comment (michaelfig) [reply packages/SwingSet/src/controller.js:216]: I've audited it (and it's tiny).  Any communications channel is only that which is provided by the application.  The `anylogger-agoric.js` files don't provide anything to an instantiated `log` except access to a subset of `console`.

File: packages/cosmic-swingset/lib/ag-solo/web.js (hunk 8 above)
Comment (michaelfig) [reply packages/cosmic-swingset/lib/ag-solo/web.js:159]: This happens after the `close` event, so the only reason to keep it around is for the `CTP_CLOSE` not to complain that the point doesn't exist.
